engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -420,12 +420,22 @@ public void setRequest(LogicalPlanRequest request)
   {
     private String containerId;
     private String errorMessage;
+    private String logFileName;
```

Comment:
You may still provide individual getter and setter if necessary, but in general, even that should not be necessary. BeanUtils should be able to handle nested objects with proper getter and setter.